data/hai/hai reveal 1 intro.txt

```diff
@@ -177,15 +178,15 @@ mission "ZZ Event Hai Secret Leaks: RSHU"
 		event "news: hai video clip published"
 		conversation "hai reveal republic or syndicate news"
 	on visit
-		dialog "You should first visit Allhome, where the most humans are, to see how things are going before you get more heavily involved."
+		dialog "You should first visit Allhome, where most of the humans in Hai space live, to see how things are going before you get more heavily involved."
```

Comment:
Different meaning:
The first one means that Allhome simply has the largest population of humans.
Your change means that Allhome explicitly has >50% of humans in Hai space.

Those are different and it's not a change I like.